src/ophys_etl/modules/decrosstalk/__main__.py

```diff
@@ -58,10 +71,34 @@ def run(self):
                 neuropil_fname = output_schema['output_neuropil_trace_file']
                 p0['output_neuropil_trace_file'] = neuropil_fname
                 (flags_0,
-                 traces_0) = run_decrosstalk(plane_0,
-                                             plane_1,
-                                             cache_dir=cache_dir,
-                                             clobber=clobber)
+                 (raw_traces,
+                  invalid_raw_traces),
+                 (unmixed_traces,
+                  invalid_unmixed_traces),
+                 (raw_trace_events,
+                  invalid_raw_trace_events),
+                 (unmixed_trace_events,
+                  invalid_unmixed_trace_events)) = run_decrosstalk(plane_0,
+                                                                   plane_1,
+                                                                   cache_dir=cache_dir,  # noqa: E501
+                                                                   clobber=clobber)  # noqa: E501
```

Comment:
this persistent `noqa` strategy doesn't really meet the spirit of saying we're using the pep8 standard. There are other ways to make these meet the standard, rather than going against the standard and having the linter ignore them,

Reply:
I find that whenever I try to accommodate the linter, I end up shortening variable/method names to something less informative. I personally feel more strongly in favor of informative naming than I do in favor of keeping lines within 80 characters. If you feel strongly about the 80 character limit, I can see what I can do.

(I would like to be able to `noqa` my way out of E129 "visual indent is at same level as next logical line"; `if (` is four characters, meaning any time you have to wrap a conditional around two lines, you end up violating that one)

Reply:
I agree, I think we should not sacrifice variable names, but there are many other ways to be compliant, one for example:
```
def run_decrosstalk(a, b, cache_dir=None, clobber=None):
    return 1, 2


def myfun():
    plane_0 = None
    plane_1 = None
    cache_dir = None
    clobber = None
    if True:
        if True:
            if True:
                if True:
                    (unmixed_trace_events,
                     invalid_unmixed_trace_events) = run_decrosstalk(
                        plane_0, plane_1, cache_dir=cache_dir,
                        clobber=clobber)
```
I don't feel strongly about the line width, I feel strongly about adhering to the standard (the very widespread standard) that we say we are going to adhere to. I'm not very interested in having this conversation over and over again about how individuals prefer their code to look. We adopted the standard, let's move on.

Reply:
I'll take another stab at it

Reply:
I believe I have removed all of my invocations of `noqa`